src/data/sql/brp/create.inschrijvingen.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE brp_prep.inschrijvingen AS
+
+  SELECT
+    pers."BSN"::varchar                                                    AS burgerservicenummer,
+    pers."Anummer"::varchar                                                AS anummer,
+    NULL::text::date                                                       AS datum_ingang_blokkering_pL,
+    NULL::text::date                                                       AS datum_opschorting_bijhouding,
+    NULL::text::varchar                                                    AS omschrijving_reden_opschorting_bijhouding,
+    CASE -- datum eerste inschrijving
+      WHEN pers."DatumInschrijving" IS NULL THEN NULL
+      WHEN pers."DatumInschrijving" = '0' THEN '0000-00-00'
+      WHEN length(pers."DatumInschrijving") = 8
+        AND pers."DatumInschrijving" != '00000000' THEN CONCAT(
```

Comment:
When is DatumInschrijving "00000000"? And why do we return "00000000" in that case?

Reply:
Scherp!
fixed, and returns 0000-00-00.

Reply:
See other comment. No need to check for "00000000" or != "00000000", as "00000000" has length 8 and will return "0000-00-00" :)

Reply:
Fixed!